packages/excalidraw/tests/__snapshots__/history.test.tsx.snap

```diff
@@ -282,45 +288,61 @@ History {
             "deleted": {
               "endBinding": {
                 "elementId": "id162",
+                "fixedPoint": [
```

Comment:
Could we describe somewhere near the definition of "fixedPoint" very briefly what is it for and why do we now need it on all arrow (now just elbow arrows)?

Reply:
I documented it in the `PointBinding` type declaration. I left it required for all linear elements because David mentioned multiple times that fixed point binding will be relevant to simple arrows and possibly linear elements as well.